### PR TITLE
docs: adds openapi reference for service api

### PR DESCRIPTION
**Pull request by andrestc**
Continues the work started by #1892

### CODE REVIEW

File: docs/reference/service_api.yaml [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+        required: true
+        schema:
+          type: string
+    get:
+      summary: Service instance additional information
+      description: |
+        The service endpoint returns additional information about the instance
+      responses:
+        404:
+          description: No additional information available
```
Comment (cezarsa) [inline docs/reference/service_api.yaml]: Doesn't 404 here mean instance not found?

File: docs/reference/service_api.yaml [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+        app-host:
+          type: string
+          description: App host
+        app-name:
+          type: string
+          description: App name
+      required:
+        - app-host
+        - app-name
+    InstanceInfo:
```
Comment (cezarsa) [inline docs/reference/service_api.yaml]: Taking a quick look at the code it looks like the InstanceInfo object is actually a map[string]string, do we really only accept `label` and `value` keys?

File: docs/reference/service_api.yaml (hunk 1 above)
Comment (andrestc) [reply docs/reference/service_api.yaml]: I don't think so. Ref: https://docs.tsuru.io/stable/services/api.html#additional-info-about-an-instance
`404: when the API doesn’t have extra info about the service instance. There’s no need to include anything in the response body.`
Also: https://github.com/tsuru/tsuru/blob/master/service/endpoint.go#L371

File: docs/reference/service_api.yaml (hunk 2 above)
Comment (andrestc) [reply docs/reference/service_api.yaml]: Yeah, I got misled by the docs: https://docs.tsuru.io/stable/services/api.html#additional-info-about-an-instance
I'll fix this.

File: docs/reference/service_api.yaml (hunk 1 above)
Comment (cezarsa) [reply docs/reference/service_api.yaml]: Ugh okay, if this is what the docs say let's keep it like this. But without this assurance we have no way to check if a given instance exists in the service. :(

File: docs/reference/service_api.yaml [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+          description: An array of plans
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Plan'
+        default:
+          $ref: '#/components/schemas/Error'
+  /resorces:
```
Comment (cezarsa) [inline docs/reference/service_api.yaml]: Typo: `resoUrces`

**Pull request merged**